astro/src/components/get-started/GitPodStart.astro

```diff
@@ -0,0 +1,66 @@
+---
+import Aside from 'src/components/Aside.astro';
+import Code from 'astro/components/Code.astro';
+import { RemoteValue } from '@fusionauth/astro-components';
+
+export interface Props {
+  /**
+   * Full URI to extract values from kickstart.json
+   * e.g. https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-ruby-on-rails-web/main/kickstart/kickstart.json
+   */
+  kickstartUri: string;
+
+  /**
+   * Flag that indicates whether this in API quickstart or not (default)
+   */
+  api?: boolean;
+}
+
+const { kickstartUri, api = false } = Astro.props as Props;
+---
+<h2> Install/Run in GitPod</h2>
+<dd>
+<a href="https://gitpod.io/#https://github.com/synedra/fusionauth-example-express-start-here" alt="Open in GitPod"><img src="https://gitpod.io/button/open-in-gitpod.svg" /></a>
+
+<p>The system will spin up a docker environment with a FusionAuth server and its components.</p>
+
+<p>Here you are using a bootstrapping feature of FusionAuth called <a href="/docs/get-started/download-and-install/development/kickstart">Kickstart</a>. When FusionAuth comes up for the first time, it will look at the <code>kickstart/kickstart.json</code> file and configure FusionAuth to your specified state.</p>
+
+<p>After the servers start up (which takes a few minutes), you will be presented with a login screen in the GitPod 'Simple Browser'.</p>
```

Comment:
gitpod timed out. we should document that so that people know it might happen.

Reply:
Will document this with an Aside in the GitPod instructions